Formula/rex.rb

```diff
@@ -206,9 +214,16 @@ class Rex < Formula
   end
 
   def install
+    # First prevent trying to sniff around elsewhere for required modules.
+    ENV.delete("PERL5LIB")
+
     ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
     ENV.prepend_path "PERL5LIB", libexec/"lib"
 
+    # If you're using a new enough perl it'll blow up & require your
+    # confirmation that you know what you're doing. Skip that.
+    ENV["PERL_CANARY_STABILITY_NOPROMPT"] = "1"
```

Comment:
@ilovezfs You should probably read [the exact wording shown to users](https://pastebin.com/PvAe04rL). Not only does it fill an entire screen, the build pauses until the user presses <kbd>return</kbd>, since the line asking *"Continue? [y ]"* is hidden for some reason. Really poor UX.

Moreover, I don't believe the `rex` author has anything to do with this: the Canary warning is being used by the `JSON::XS` module (which was developed by the author of the long-winded message itself).